Modules/Venice/Tests/VeniceTests/Venice/TickerTests.swift

```diff
@@ -3,11 +3,16 @@ import XCTest
 
 public class TickerTests : XCTestCase {
     func testTicker() {
-        let ticker = Ticker(period: 10.milliseconds)
+        let tickerPeriod = 20.milliseconds
```

Comment:
We should probably increase the period (curr 20ms) and time tolerance (curr 10ms) because 20 +/- 10 ms make the macOS tests fail on Travis. I guess the VM is too loaded to respect such short deadlines. We should be good with a 2.5x factor. I'll PR